src/tools/vtapp/VTApp.csproj

```diff
@@ -35,6 +35,24 @@
     <Optimize>true</Optimize>
     <DefineConstants>TRACE</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|ARM64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|Win32'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|ARM64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|Win32'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>

```

Comment:
Why does this get a `<Prefer32Bit>` but not the other .csproj?

Reply:
> You added a UTF-8 BOM to some files. Is that intentional or Visual Studio's unfortunate doing (I'm assuming it's the latter)? We usually keep all files in this project without BOM.

Correct.  VS did it and I was not easily able to save without the {BOM}.  What's the easiest way to remove it without resorting to a binary editor?

Reply:
@dmachaj I usually use vim (`:set nobomb | w!`), but fwiw If you enabled "allow maintainers to push changes..." I'd be happy to fix this up for you